server/common/HatoholArmPluginInterface.cc

```diff
@@ -157,6 +157,10 @@ struct HatoholArmPluginInterface::Impl {
 		connected = false;
 		hapi->onSessionChanged(NULL);
 	}
+	
+	bool connectstat(void) {
+		return connected;
```

Comment:
Yes, I understand.
"connected" will be changed from other threads.
Therefore, lock(connectionLock) is required.